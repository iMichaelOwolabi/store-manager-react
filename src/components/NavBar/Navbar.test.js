import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '.';

describe('<Narbar Component />', () => {
  it('should', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
