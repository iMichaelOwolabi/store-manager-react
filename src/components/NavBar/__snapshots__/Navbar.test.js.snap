// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Narbar Component /> should 1`] = `
<header>
  <ToastContainer
    autoClose={5000}
    bodyClassName={null}
    className={null}
    closeButton={
      <CloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="top-right"
    progressClassName={null}
    progressStyle={null}
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <div
    className="container"
  >
    <div
      id="logo"
    >
      <h1>
        Store Manager
      </h1>
    </div>
  </div>
</header>
`;
